src/porepy/fracs/meshing.py

```diff
@@ -83,6 +83,10 @@ def subdomains_to_mdg(
     # will be added to the mixed-dimensional grid.
     create_interfaces(mdg, node_pairs)
 
+    # We can also add boundary grids to the MixedDimensionalGrid after having split
```

Comment:
I don't understand the comment. Is the point the order of operations, i.e., boundaries AFTER fractures?